pkg/uploader/events.go

```diff
@@ -0,0 +1,13 @@
+package uploader
+
+import (
+	"context"
+	"io"
+
+	up "github.com/openshift/assisted-service/internal/uploader"
+	"github.com/openshift/assisted-service/pkg/uploader/models"
+)
+
+func ExtractEvents(ctx context.Context, r io.Reader) ([]models.Events, error) {
+	return up.ExtractEvents(r)
```

Comment:
Why aliasing this pkg instead of declaring the code in `pkg`?

Reply:
IMHO I think it's better to keep the code that creates the payload next to the code that parses the payload.

If I started to move both of them to pkg/, then some events would be created in internal/ and some in pkg/ (& I will have a dependency cycle).

Do you suggest to move everything at once ?
WDYT ?